topic5/test/faucet.test.js

```diff
@@ -0,0 +1,72 @@
+const Faucet = artifacts.require('../contracts/Faucet.sol');
+
+const testUtil = require('./utils/test.util.js');
+
+contract('Faucet', (accounts) => {
```

Comment:
You can destruct the accounts to variables directly like this - instead `(accounts) =>`  - `([owner, receiver, sender, exchange, other]) =>` and you have those variables assigned with the addresses at the current indices.